buildkite/bazelci.py

```diff
@@ -1170,6 +1170,12 @@ def use_bazelisk_migrate():
     """
     return is_trueish(os.environ.get("USE_BAZELISK_MIGRATE"))
 
+def is_downstream_pipeline():
+    """
+    Return true if BAZELCI_DOWNSTREAM_PIPELINE is set
```

Comment:
I have manually set `BAZELCI_DOWNSTREAM_PIPELINE=true` for all downstream pipelines.